.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
note: There's a forms feature in Beta. Instead of a single Markdown text field users can just fill out fields and select from dropdowns.

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

Reply:
yep, great call - I also had this on my radar and was going to attempt converting to v2 templates, which leverage forms.
i'll follow on as an iteration with that